.travis.yml

```diff
@@ -8,8 +8,27 @@ perl:
   - "5.14"
   - "5.12"
   - "5.10"
+matrix:
+  include:
+    - perl: 5.22
+      env: COVERAGE=1   # enables coverage+coveralls reporting
+      include:
+    - perl: 5.24
+      env: COVERAGE=1   # enables coverage+coveralls reporting
+sudo: false
 before_install:
   - git clone git://github.com/travis-perl/helpers ~/travis-perl-helpers
   - source ~/travis-perl-helpers/init
   - build-perl
   - perl -V
+  - build-dist
+  - cd $BUILD_DIR             # $BUILD_DIR is set by the build-dist command
+install:
+  - cpan-install ExtUtils::MakeMaker~6.68 --deps
+  - cpan-install --coverage
+before_script:
+  -  coverage-setup
+script:
+  - prove -l -j$(test-jobs) $(test-files)   # parallel testing
+after_success:
+  - coverage-report
```

Comment:
Better not to. Took me a while to debug the file. What that file does is to create a helper dir and update it. Not too fond of these shell scripts. Anyway, we need prove and we need coverage report, so I will leave these one as they are.